lib/ebnf.mjs

```diff
@@ -4,11 +4,12 @@ import visitor from "./visitor.mjs";
 import serialize from "./serializer.mjs";
 
 export default
-function ebnf(source) {
+function ebnf(source, min_name_length = 26) {
```

Comment:
Is this just to keep the diff small? Do we intend to keep this?

Reply:
Yes, it's to minimize the diff. I don't mind leaving it here and giving the EBNF some breathing air. I also don't mind removing this change and using `git diff --ignore-space-change` (a.k.a. `git diff -b`) to view changes between syntax versions. Even GitHub has this option now, so maybe that's the way to go?